packages/apm/src/integrations/tracing.ts

```diff
@@ -460,6 +453,7 @@ export class Tracing implements Integration {
       logger.log('[Tracing] finishIdleTransaction', active.transaction);
       // true = use timestamp of last span
       active.finish(true);
+      Tracing._resetActiveTransaction();
```

Comment:
Since we didn't "unset" the transction (clean up) after it flushed, it produced a weird debug output if you switched tabs that the active transaction was "unset" because of page going to background.